src/components/connection-modal/connection-modal.css

```diff
@@ -364,20 +364,25 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding-bottom: 12px;
+    padding-top: 1rem;
+    padding-bottom: .75rem;
+    padding-left: .75rem;
+    padding-right: .75rem;
+}
+
+.bottom-area .bottom-area-item+.bottom-area-item {
```

Comment:
css "+" symbol means:

"a+b" == "style for b, when it follows a"

so, this style will apply to every bottom-area-item *except* the first one.